### PR TITLE
All: fix some broken links

### PR DESCRIPTION
**Pull request by arthurvr**
Fixes a bunch of broken links pointed out by our spider.

### CODE REVIEW

File: entries/id-selector.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
...
     <p>Calling <code>jQuery()</code> (or <code>$()</code>) with an id selector as its argument will return a jQuery object containing a collection of either zero or one DOM element.</p>
     <p>Each <code>id</code> value must be used only once within a document. If more than one element has been assigned the same ID, queries that use that ID will only select the first matched element in the DOM. This behavior should not be relied on, however; a document with more than one element using the same ID is invalid.</p>
-    <p>If the id contains characters like periods or colons you have to <a href="http://learn.jquery.com/faq/how-do-i-select-an-element-by-an-id-that-has-characters-used-in-css-notation/">escape those characters with backslashes</a>.</p>
+    <p>If the id contains characters like periods or colons you have to <a href="http://stage.learn.jquery.com/using-jquery-core/faq/how-do-i-select-an-element-by-an-id-that-has-characters-used-in-css-notation/">escape those characters with backslashes</a>.</p>
```
Comment (scottgonzalez) [inline entries/id-selector.xml]: Definitely not a link to stage.

Comment (arthurvr) [inline entries/id-selector.xml]: Agrrr of course. Updating.

File: pages/Types.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 </style>
 <p>JavaScript provides several built-in datatypes. In addition to those, this page documents virtual types like Selectors, enhanced pseudo-types like Events and all and everything you wanted to know about Functions.
 </p>
-<p>You should be able to try out most of the examples below by just copying them to your browser's JavaScript Console (Chrome, Safari with Develop menu activated, IE 8+) or <a href="http://www.getfirebug.com/">Firebug</a> console (<a href="http://www.getfirefox.com">Firefox</a>).
+<p>You should be able to try out most of the examples below by just copying them to your browser's JavaScript Console (Chrome, Safari with Develop menu activated, IE 8+) or <a href="http://www.getfirebug.com/">Firebug</a> console (<a href="https://www.mozilla.org/en-US/firefox/new/?utm_source=getfirefox-com&utm_medium=referral">Firefox</a>).
```
Comment (scottgonzalez) [inline pages/Types.html]: Remove the query.

Comment (arthurvr) [inline pages/Types.html]: fixed

**Pull request closed without merging**